src/Stratis.Features.FederatedPeg/Collateral/CollateralChecker.cs

```diff
@@ -134,14 +128,13 @@ private async Task UpdateCollateralInfoContinuouslyAsync()
                     return;
                 }
 
-                bool success = await this.UpdateCollateralInfoAsync(this.cancellationSource.Token).ConfigureAwait(false);
-
-                if (!success)
-                    this.logger.LogWarning("Failed to update collateral. Ensure that mainnet gateway node is running and API is enabled.");
+                bool collateralUpdated = await this.UpdateCollateralInfoAsync(this.cancellationSource.Token).ConfigureAwait(false);
```

Comment:
collateralUpdated can be set in `UpdateCollateralInfoAsync`

Reply:
Ok yeah if we do this then we need to check it in the while loop on initialization.